app/views/teams/show.html.erb

```diff
@@ -12,28 +12,22 @@
 </div>
 
 <div class="row teamCenter">
-    <% @team.users.each do |user| %>
-      <div class="small-3 column">
-        First Name:
-        <br />
-        <%= user.first_name %>
-      </div>
-      <div class="small-3 column">
-        Last Name:
-        <br />
-        <%= user.last_name %>
-      </div>
-      <div class="small-3 column">
-        Email:
-        <br />
+  <% @team.users.each do |user| %>
+    <div class="small-6 column">
+      Email:
+      <br />
```

Comment:
Did the text not break just by using the <p> tag? You can also consider wrapping "Email:" in span and then styling the appropriate margin below it.